### ISSUE TITLE
Restore Purchase Button

### ISSUE THREAD

**Issue by michaeldugan** [djangdal/OnePlusOne#1]
The app will need a restore purchase button or else it doesn't meet Apple's guidelines. Let me know if you need help adding one :)

**Comment by djangdal** [comment]
Hi, yes. that is absolutely correct :) I have not implemented it but if you want, you are welcome to help out.

**Comment by michaeldugan** [comment]
I am working on the code right now. But I usually do it when there's already a button on the storyboard.

**Comment by QuetzalPretzel** [comment]
What out @djangdal Michael/Daniel Dugan (whichever is his name) is trying to clone your app.  If you make the changes for him he will finish submitting your app to the App Store.  He's a serial cloner, see here: https://itunes.apple.com/us/developer/daniel-dugan/id860196910 all these are clones, e.g..  "Asteroid Evaders" = "Space Evaders", "Ninja Thrower" = "Ninja Blocks".

Don't take my work for it, see what hawkfalcon a victim of his cloning has to say about his apps: https://github.com/hawkfalcon/SpaceEvaders/issues/8

**Comment by michaeldugan** [comment]
@QuetzalPretzel Yeah your real tough. I have not had any intention of cloning this app. I have just given him advice on how too fix his app. So stop calling me a serial cloner because I am not. I don't think you realize that if apps are not licensed using open source is not cloning. So why don't you leave me alone and stay out of my business.

**Comment by michaeldugan** [comment]
@djangdal I think I may have a solution to the restore purchase problem. Do you want me to create a new repo or contribute it directly to this one?

**Comment by djangdal** [comment]
@michaeldugan due to your bad reputation I would like you not to contribute to the repository. It's not a big problem and I can do i t myself
